docs/configuration/kubeedge.md

```diff
@@ -148,7 +148,7 @@ kubectl apply -f ~/kubeedge/yaml/node.json
 
 RootCA certificate and a cert/key pair is required to have a setup for KubeEdge. Same cert/key pair can be used in both cloud and edge.
 
-cert/key should exist in /etc/kubeedge/ca and /etc/kubeedge/certs. Otherwise please refer to [generate certs](https://github.com/kubeedge/kubeedge/blob/master/docs/setup/kubeedge_install_source.md#generate-certificates) to generate them.
+cert/key should exist in /etc/kubeedge/ca and /etc/kubeedge/certs. Otherwise please refer to [generate certs](https://github.com/kubeedge/kubeedge/blob/release-1.3/docs/setup/kubeedge_install_source.md#generate-certificates) to generate them.
```

Comment:
Better to use master branch here

Reply:
I think master branch is not a good choice since they are deleted now.